mods/ts/rules/defaults.yaml

```diff
@@ -823,7 +824,6 @@
 	Targetable@AIRBORNE:
 		TargetTypes: Air
 		RequiresCondition: airborne
-	Selectable:
```

Comment:
Is this correct? Travis shows some errors that could be related.

Reply:
No, that was not intentional. I did not notice it in the diff. Fixed.